wlauto/workloads/powerpoint/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -50,24 +49,30 @@ public void runUiAutomation() throws Exception {
         confirmAccess();
         skipSignInView();
 
-        if (testType.equals("create")) {
-            newPresentation();
-            selectTemplate(templateName);
-            dismissToolTip();
-            editTitle(titleText);
-            clickNewSlide();
-            setSlideLayout();
-            addImage();
-            clickThumbNail(0); // go back to first slide before presenting
-            presentSlides(2);
-        }
+        // ----------------------------------------------------------------
+        // Create presentation from template
+        // ----------------------------------------------------------------
+        newPresentation();
+        selectTemplate(templateName);
+        dismissToolTip();
+        editTitle(titleText);
+        clickNewSlide();
+        setSlideLayout();
+        addImage();
+        clickThumbNail(0); // go back to first slide before presenting
+        presentSlides(2, "None");
+
+        // Return to main application screen
+        pressBack();
+        pressBack();
 
-        if (testType.equals("load")) {
```

Comment:
As discussed during the conf call, we should instead allow the agenda to pass a param called 'use_existing_doc' to the workload so that we can enable this part of the test when required.   Please come up with a better param name with Abe and start using it for the three MS workloads